src/plugins/intel_gpu/src/kernel_selector/kernels/gemm/gemm_kernel_tiled_opt.cpp

```diff
@@ -213,6 +213,19 @@ JitConstants GemmKernelTiledOpt::GetJitConstants(const gemm_params& params) cons
             jit.AddConstant(MakeJitConstant("HAS_DYNAMIC_K_PADDING", 1));
         if (has_dynamic_n_padding)
             jit.AddConstant(MakeJitConstant("HAS_DYNAMIC_N_PADDING", 1));
+
+        auto hasDynamicPad = [](DataTensor dt) -> bool {
+            auto dims = dt.GetDims();
+            for (auto d : dims) {
+                if (d.pad.is_dynamic)
+                    return true;
+            }
+            return false;
+        };
+        if (hasDynamicPad(params.inputs[0]))
+            jit.AddConstant(MakeJitConstant("INPUT0_HAS_PADDING", 1));
+        if (hasDynamicPad(params.inputs[1]))
+            jit.AddConstant(MakeJitConstant("INPUT1_HAS_PADDING", 1));
```

Comment:
These flags are for static padding. Please use either of DYNAMIC_K or DYNAMIC_N padding

Reply:
"has_dynamic_n_padding" false, but still input1 has some padding which requires offset calculation from the padded. Maybe I can add new constant for this situation instead of using "INPUT1_HAS_PADDING"

Reply:
Input1's padding can either of dynamic_n_padding or dynamic_k_padding or static padding (INPUT1_HAS_PADDING). What is the unclear case?

Reply:
Fixed to use "HAS_DYNAMIC_K_PADDING" insted of INPUT1_HAS_PADDING